src/main/resources/application.properties

```diff
@@ -1 +1,9 @@
-
+server.port=8080
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.datasource.username=postgres
+spring.datasource.password=2000
+spring.jpa.show-sql=true
+spring.jpa.generate-ddl=false
```

Comment:
read about liquibase, and try make tables in database using it, and turn off ddloptions please